test/Be.Vlaanderen.Basisregisters.Shaperon.Tests/DbaseIntegerDigitsTests.cs

```diff
@@ -74,14 +74,22 @@ public void ToInt32ReturnsExpectedValue()
         [Fact]
         public void ToLengthReturnsExpectedValue()
         {
-            var value = _fixture.Create<int>().AsDbaseIntegerDigitsValue();
+            var value = _fixture.Create<int>().AsDbaseFieldCompatibleDbaseIntegerDigitsValue();
             var sut = new DbaseIntegerDigits(value);
 
             var result = sut.ToLength();
 
             Assert.Equal(new DbaseFieldLength(sut.ToInt32()), result);
         }
 
+        [Fact]
+        public void ToLengthThrowsWhenNoDigits()
```

Comment:
Made the exception explicit for now. Not sure where and if this method should even exist since it's easy to shoot yourself in the foot with.